src/Diagnostics.ModelsAndUtils/Models/Response.cs

```diff
@@ -33,6 +33,11 @@ public class Response
         /// </summary>
         public List<Insight> Insights { get; set; }
 
+        /// <summary>
+        /// ASC Insight List
+        /// </summary>
+        public List<AzureSupportCenterInsight> AscInsights { get; set; }
```

Comment:
having this outside DiagnosticData means it will always show up on top irrespective where developer is writing it?

Reply:
No, these are not actually returned from the API. For insights calls I return them, otherwise I add an insight to diagnostic data.